### ISSUE TITLE
Loss is constantly 0 while evaluating the clustering performance

### ISSUE THREAD

**Issue by JuOllie** [Tony607/Keras_Deep_Clustering#1]
Hey!
First off, thank you so much for sharing your code- it's super helpful :)

I wanted to know why the loss is 0 for all the iterations, during the evaluation stage, and if there is any implication regarding the same.

**Comment by hxd1011** [comment]
> Hey!
> First off, thank you so much for sharing your code- it's super helpful :)
> 
> I wanted to know why the loss is 0 for all the iterations, during the evaluation stage, and if there is any implication regarding the same.

check https://github.com/XifengGuo/DEC-keras/blob/master/DEC.py

line 249

**Comment by Tony607** [comment]
Thanks for commenting, the loss now updates during the training.
